docs/guide/2-user-guide/1-installation/index.md

```diff
@@ -95,10 +95,13 @@ If MPI support is enabled (the default), the C API is automatically built.
 
 ### Workaround issue in HIP
 
-HIP is a relatively new language, and some issues still need to be ironed out. As a workaround to an [issue](https://github.com/ROCm-Developer-Tools/HIP/pull/1543) in HIP's JIT infrastructure, please set the following if you've built HIP from source:
+For custom installs of HIP 3.9.0 and above, some paths have to be configured to ensure the JIT compiler can locate the HIP runtime and compiler tools
 
 ```bash
-    export HIP_PATH=/opt/rocm/hip
+    export ROCM_PATH=/path/to/hip-3.9.0
+    export HIP_PATH=$ROCM_PATH
+    export LLVM_PATH=/path/to/llvm-amdgpu-3.9.0
+    export HIP_DEVICE_LIB_PATH=/path/to/rocm-device-libs-3.9.0/amdgcn/bitcode
```

Comment:
Would it be possible and make sense to get defaults for those at build time (from the build environment) and set them internally, but keep them overrideable with env vars?

Reply:
Hm yeah, not sure, cause you'd end up with absolute paths in binaries, which is maybe not what you want when you distribute binaries. If rocm is installed by apt/yum/.. it should not be necessary to set these variables, I can clarify that a bit more.

Reply:
Might be worth to document it, yes please.
This is the kind of the stuff people randomly stick in their scripts and carry around forever until it starts to break something else...
What happens btw if those vars are not set? Do we have a FAQ somewhere such that people find it when googling the error messages?

Reply:
I'll address this in a separate pr, don't have access to AMD GPUs right now.